onnxruntime/python/tools/transformers/convert_to_onnx.py

```diff
@@ -100,8 +100,16 @@ def parse_arguments():
     parser.add_argument('-e', '--use_external_data_format', required=False, action='store_true')
     parser.set_defaults(use_external_data_format=False)
 
-    parser.add_argument('--batch_size', required=False, type=int, default=1, help='Batch size for GPT model with beam search')
-    parser.add_argument('--beam_size', required=False, type=int, default=4, help='Beam size for beam search')
+    parser.add_argument('--beam_size', type=int, default=4, help='Beam size if greedy/top-p/top-k sampling is needed')
+    parser.add_argument('--ignore_eos', type=bool, default=False, help='If ignore end of sentence token in model inference (one step beam search with configuration only).')
```

Comment:
It might be help to use argument groups to replace "(one step beam search with configuration only)", see [example](https://www.kite.com/python/answers/how-to-group-argparse-arguments-in-python)

Reply:
Good suggestion! I have added argument groups. Thanks!

Reply:
Good suggestion! Done